futures-core/src/future.rs

```diff
@@ -9,6 +9,11 @@ pub use core::future::Future;
 
 /// An owned dynamically typed [`Future`] for use in cases where you can't
 /// statically type your result or need to add some indirection.
+///
```

Comment:
This follows the wording and pattern in std, for example https://doc.rust-lang.org/std/iter/struct.Cloned.html

> This struct is created by the [cloned](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.cloned) method on [Iterator](https://doc.rust-lang.org/std/iter/trait.Iterator.html). See its documentation for more.

Here is how it looks rendered: 
<img width="1107" alt="Screenshot 2024-10-03 at 9 34 52 AM" src="https://github.com/user-attachments/assets/81259caf-2229-474c-a352-5460a1cb892f">